benchmarks_test.go

```diff
@@ -11,8 +12,13 @@ import (
 	"code.cloudfoundry.org/go-diodes"
 )
 
+var (
+	randData func(int) *[]byte
+)
+
 func TestMain(m *testing.M) {
 	log.SetOutput(ioutil.Discard)
+	randData = randDataGen()
```

Comment:
This can be invoked as part of the init process via:

`var randData = randDataGen()`

I know you love `TestMain` but the above makes more sense to me.